app/ts/McdaApp/Workspace/CurrentTab/Preferences/EquivalentChange/EquivalentChangeStatement/EquivalentChangeRangeStatement.tsx

```diff
@@ -28,6 +28,7 @@ export default function TradeOffReferenceCriterionStatement(): JSX.Element {
     setDialogOpen(false);
     setAnchorElement(null);
   }
+  const unit = referenceCriterion.dataSources[0].unitOfMeasurement;
 
   return _.isNumber(referenceValueFrom) && _.isNumber(referenceValueTo) ? (
```

Comment:
Are these checks still needed? Since we only initialize the context if it is possible to set these values.